src/queries/hooks/rewards/useGenerateUserId.ts

```diff
@@ -0,0 +1,9 @@
+import { useMutation } from '@tanstack/react-query';
+import { generateUserIdByWallet } from 'queries/queryKeys';
+import { generateUserId } from 'queries/services/rewards';
+
+export const useGenerateUserId = () =>
```

Comment:
@corlard3y I don't think we need and it's similar to src/queries/hooks/rewards/useCreateUserRewardsDetails.ts. Could you please look into it.

If this is not required we can remove the hooks, services, models and their respective index.ts files.